core/src/main/scala/scalan/Entities.scala

```diff
@@ -28,7 +28,7 @@ trait Entities extends TypeDescs { self: Scalan =>
       case _ => false
     })
 
-    override def hashCode = Objects.hash(getClass, typeArgsDescs)
+    override def hashCode = getClass.hashCode() * 31 + typeArgsDescs.hashCode()
```

Comment:
Remove imports not used anymore, provide ScalaDoc for the trait

Reply:
done